aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,67 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
+
+ The follwoing instrcutions were tested using CentOS 7.
+ 
+Install Mono
+^^^^^^^^^^^^
 
 .. note::
 
-    Installation steps for CentOS, Fedora and derivatives are not currently available but should be available soon. The commands are mostly the same, with some differences to account for the different package managers used on these systems. Learn how you can `contribute <https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md>`_ on GitHub.
-  
+This step it's optional if you want only use CoreCLR.
```

Comment:
Grammatical nit pick, this should read: "This step is optional if you will only be using CoreCLR"